bucket/chkcpu.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "2.14",
+    "description": "Checks CPU Vendor, Model, Internal speed, and L1 cache mode settings.",
+    "homepage": "http://web.inter.nl.net/hcc/J.Steunebrink/chkcpu.htm",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://web.inter.nl.net/hcc/J.Steunebrink/chkcpu.htm"
+    },
+    "url": "http://web.inter.nl.net/hcc/J.Steunebrink/ckcpu214.zip",
+    "hash": "ce9142e29000f8299750a1880056a8a812c0fef91cac16b843ad947b18114d16",
+    "bin": [
+        "chkcpu32.exe",
```

Comment:
This shim is needed? I think the below one is sufficient.

Reply:
I tend to add it, because when someone reads some article that says something like:
`Run chkcpu32.exe /R`
, they can find `chkcpu32.exe` without a problem.

Reply:
Okay, makes sense 👍